### ISSUE TITLE
IOS support

### ISSUE THREAD

**Issue by NM118** [straight-tamago/misakaX#412]
When is normal IOS 18.2 support coming? I’ve been waiting for the IOS 18.2 support update for ages.

**Comment by Ali-Milani-13** [comment]
> When is normal IOS 18.2 support coming? I’ve been waiting for the IOS 18.2 support update for ages.

I don't think it will ever happen. `mobilegestalt` trick doesn't work on 18.2+. 

same as troll store. it will not work either.
